### PR TITLE
Offers without migrations

### PR DESCRIPTION
**Pull request by stepanzubkov**
I rewrote offers without alembic migrations. Work in progress.

### CODE REVIEW

**Comment by stepanzubkov** [comment]
I am going to create admin methods for offers

File: src/api/app/services/pagination.py
Comment (kirillzhosul) [inline src/api/app/services/pagination.py:1]: Looks like this file is included but not used for that PR.

File: src/api/app/serializers/offer.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "text": offer.text,
+    }
+    if offer["user_id"]:
+        offer["user_id"] = offer.user_id
+    else:
+        offer.pop("user_id")
+
+    if in_list:
+        return offer
+    return {"offer": offer}
```
Comment (kirillzhosul) [inline src/api/app/serializers/offer.py:28]: return {"offer": offer} if in_lsit else offer

File: src/api/app/routers/v1/admin.py [hunk 2]
```diff
@@ -163,3 +165,35 @@ async def method_admin_unban_user(
...
+
+
+@router.get("/_admin.getOffers")
+async def method_admin_get_offers(
+    req: Request,
+    db: Session = Depends(get_db),
+) -> JSONResponse:
+    """Returns list of offers."""
+
+    await validate_user_allowed_to_call_admin_methods(req, db)
```
Comment (kirillzhosul) [inline src/api/app/routers/v1/admin.py:177]: Later should be refactored.

File: src/api/app/database/models/offer.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    text = Column(String, nullable=False)
+    email = Column(String, nullable=False)
+    phone_number = Column(String(18), nullable=False)
+    full_name = Column(String, nullable=False)
+
+    # Time
+    time_created = Column(
+        DateTime(timezone=True), server_default=func.now(), nullable=False
+    )
+    time_updated = Column(DateTime(timezone=True), onupdate=func.now())
```
Comment (kirillzhosul) [inline src/api/app/database/models/offer.py:24]: Later should be refactor as Mixin

File: src/api/app/database/crud/offer.py
Comment (kirillzhosul) [inline src/api/app/database/crud/offer.py:1]: There is no repo for offer table, new tables should be written in repo-like CRUD.

**Review by kirillzhosul — APPROVED** [review]

**Comment by kirillzhosul** [comment]
LGTM, but those review comments should be noticed!

**Pull request merged**